api/CMakeLists.txt

```diff
@@ -13,11 +13,17 @@ install(
   LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
   ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR})
 
+set(EXCLUDE_PATTERN "")
+if(NOT WITH_METRICS_PREVIEW)
+  set(EXCLUDE_PATTERN "metrics")
```

Comment:
Just wondering this pattern could exclude some unexpected header files, like some potential internal metrics for SDK?

Reply:
Yes, if there are any header files with the name containing `metrics` under the non-metrics directory under `include/opentelemetry/sdk/`, it will exclude them, but I don't see why should someone create that.

Reply:
Could we include the full or partial path? Agree that the name conflict chance is very low and not a blocker though.

Reply:
Yes, good point. I will add that.

Reply:
ok, I did some quick tests. The current pattern will only remove files/directories with the name "metrics", and not those containing "metrics". So we should be good here.